### ISSUE TITLE
Documentation on how to use 'Translation of' function

### ISSUE THREAD

**Issue by globalcow** [saltcorn/saltcorn#1437]
There is the possibility to set that one field is a translation of another. This will add an 'localized_by' and 'localizes' attribute. How can I make use of this functionality? I missing documentation on this topic.

**Comment by AleksandrSokolov** [comment]
1) First of all you need to add language.
![image](https://user-images.githubusercontent.com/327030/202306027-e6d3ab6c-b7b9-4ae7-b965-3c0cb896cbdb.png)

2) Add original field. In my example field_to_translate

3) Add field with translation. In my example field_to_translate_bg

![image](https://user-images.githubusercontent.com/327030/202306741-5dd93848-6abc-43bb-a8ba-f1e8ef6f528d.png)

4) You will get
![image](https://user-images.githubusercontent.com/327030/202306924-d8d3ab4a-ac31-4029-888e-397cd1588208.png)

5) then you can use both fields in views
![image](https://user-images.githubusercontent.com/327030/202307781-c4d26ae2-eaa6-4cd4-ae7b-bbea614e363a.png)

**Comment by globalcow** [comment]
Hi @AleksandrSokolov , thank you for your reply!

Sorry, I still don't get it. In you example you create two fields which are related by the localization. Of course you can use both fields in the view like any other fields. But what is the advantage of adding the localization information? Where can I make use of this relation?

Sorry if this is a dumb question 😆

**Comment by AleksandrSokolov** [comment]
Hi @globalcow ,
If you will use field_to_translate in views, than
depends of you locale you will get automatic substitute of localized field into views.

**Comment by AleksandrSokolov** [comment]
@globalcow see my next example. I used other fields txt and locale ru:
![image](https://user-images.githubusercontent.com/327030/202917870-e1ad0c86-9897-4530-8425-8706eb84da31.png)

![image](https://user-images.githubusercontent.com/327030/202917891-df0f4cb8-3988-4e9d-bde0-8c3eb0fc71be.png)

![image](https://user-images.githubusercontent.com/327030/202917915-c5b806a6-22b8-4376-818d-e1111d9668e1.png)

![image](https://user-images.githubusercontent.com/327030/202917935-fee23365-7698-4561-a2fa-e114974aa719.png)

**Comment by AleksandrSokolov** [comment]
So this is usefull

**Comment by globalcow** [comment]
Hi @AleksandrSokolov, great, thank you for the explanation!

I expected it to work like that but I always tried to use it as "Summary field" for a foreign key table field. I selected a 'localized_by' field in the "Summary field" selection. However, if this summary field is then used in a field view or a dropdown, it is not translated automatically. I think that would be a really nice addition?! What is your opinion?

**Comment by glutamate** [comment]
@globalcow you're right that the summary field is not currently translatable. In many places you can give a formula instead of the summary field (e.g. filters or select key value in edit). Maybe we should look at introducing `lang` ad a variable in formulas like @j2l suggested

**Comment by globalcow** [comment]
Hi @glutamate , thank you for your reply. Yes, in the moment I use the label formula in the select field to use the right column for the current language. But then I have to build a view for each language. This could be solved with the `lang` variable. 
But I also like the translatable summary field. It would be more automatic and I think it would work for array aggregation with a many to many relation, too. I haven't found a workaround for this and had to switch to the batch show view. However, then again I need a badge view for each language.

**Comment by globalcow** [comment]
#1649
